code/game/objects/items/weapons/flamethrower.dm

```diff
@@ -60,10 +60,17 @@
 		item_state = "flamethrower_0"
 	return
 
-/obj/item/weapon/flamethrower/afterattack(atom/target, mob/user, flag)
-	if(flag) return // too close
-	// Make sure our user is still holding us
-	if(user && user.get_active_held_item() == src)
+/obj/item/weapon/flamethrower/afterattack(atom/target, var/mob/living/carbon/human/user, flag)
+	if(flag) return // too close  
```

Comment:
Add an extra line for the return here(i know it was like this before, but change while you are at it)!